redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/EnablementArea.tsx

```diff
@@ -25,7 +26,13 @@ export interface Props {
   guides: Record<string, IEnablementAreaItem>
   tutorials: Record<string, IEnablementAreaItem>
   loading: boolean
-  openScript: (script: string, path?: string, name?: string) => void
+  openScript: (
```

Comment:
if 5 arguments exists, I think should refact to 1 object argument `{...}`

Reply:
fixed